File: website/src/interfaces/City.ts
import { Neighborhood } from './Neighborhood';

export interface City {
  ID: number;

  Name: string

  Population: number;

  FeaturedImage: string;

  MetroID: number;

  Notes: string;
}

export interface DetailedCity extends City {
  Pics: string[]

  Neighborhoods: Neighborhood[]
}

File: java-spring/src/main/java/com/adhiravishankar/citiesorganizer/models/City.java
package com.adhiravishankar.citiesorganizer.models;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.Field;

@Document("cities")
public class City {

    @Id
    String id;

    @Field("metro_id")
    String MetroID;

    @Field("name")
    String Name;

    @Field("population")
    String Population;

    @Field("featured_images")
    String FeaturedImage;

    @Field("notes")
    String Notes;


}
